src/include/kdbprivate.h

```diff
@@ -518,10 +518,13 @@ int elektraValidateKeyName(const char *name, size_t size);
  */
 #define ELEKTRA_MAX_ARRAY_SIZE (21)
 
+#if !defined(__cplusplus)
```

Comment:
Removing the `#if !defined(__cplusplu)` line (and the corresponding `#endif`) results in 

	Linking CXX shared module ../../../lib/libelektra-dump.so
	[ 40%] Built target elektra-dump
	In file included from /Users/pnirschl/Documents/libelektra/src/libtools/src/plugin.cpp:16:
	/Users/pnirschl/Documents/libelektra/src/include/kdbprivate.h:525:50: error: unknown type name 'kdb_long_long_t'; did you mean 'kdb::long_long_t'?
	int elektraReadArrayNumber(const char *baseName, kdb_long_long_t *oldIndex);
	                                                 ^~~~~~~~~~~~~~~
	                                                 kdb::long_long_t
	/Users/pnirschl/Documents/libelektra/src/include/kdbtypes.h:47:18: note: 'kdb::long_long_t' declared here
	        typedef int64_t long_long_t; // default: 0
	                        ^
	In file included from /Users/pnirschl/Documents/libelektra/src/libtools/src/plugin.cpp:16:
	/Users/pnirschl/Documents/libelektra/src/include/kdbprivate.h:526:44: error: unknown type name 'kdb_long_long_t'; did you mean 'kdb::long_long_t'?
	int elektraWriteArrayNumber(char *newName, kdb_long_long_t newIndex);
	                                           ^~~~~~~~~~~~~~~
	                                           kdb::long_long_t
	/Users/pnirschl/Documents/libelektra/src/include/kdbtypes.h:47:18: note: 'kdb::long_long_t' declared here
	        typedef int64_t long_long_t; // default: 0
	                        ^
	2 errors generated.
	make[2]: *** [src/libtools/src/CMakeFiles/elektratools.dir/plugin.cpp.o] Error 1
	make[1]: *** [src/libtools/src/CMakeFiles/elektratools.dir/all] Error 2
	make: *** [all] Error 2